buildscripts/kokoro/unix.sh

```diff
@@ -36,3 +37,27 @@ pushd examples
 mvn verify --batch-mode
 popd
 # TODO(zpencer): also build the GAE examples
+
+# Build release artifacts and move it to the artifacts dir
+# If we want to build but not store the artifacts, do not set 'define_artifacts' in cfg file.
+if [[ -z "${BUILD_MVN_ARTIFACTS:-}" ]]; then
+  exit 0
+fi
+
+# TODO(zpencer): if linux builds use this script, then also build for 32bit (-m32 flag)
+# Today, only macos uses this script and macos targets 64bit only
+
+# re-build protobuf using the flags listed in RELEASING.md
```

Comment:
This script seems substantially different. It seems this should be a completely different script. We don't need those gradle properties nor examples building. We just need the compiler.

Reply:
I wanted to make sure the build succeeds before trying to build the artifacts. I think I will call out to the original script before calling this one to achieve that.